cypress/integration/page.spec.js

```diff
@@ -421,7 +450,7 @@
     describe('if posts are loaded successfully', () => {
       beforeEach(() => {
         page.mockUsers();
-        page.mockUser1Posts()
+        page.mockUser1Posts();
```

Comment:
Similar to the previous point, the `page.mockUser1Posts()` call should be followed by `.as('user1PostsRequest')`.